### ISSUE TITLE
Performance based on airbert.bin

### ISSUE THREAD

**Issue by hekj** [airbert-vln/airbert-recurrentvln#3]
Hi, Shizhe
     Thanks for your valuable work.  
     I have tried to train the model on REVERIE based on airbert.bin with the below configuration:
![image](https://user-images.githubusercontent.com/17741857/138736981-f48051b7-3aea-47af-9430-4a97c4618c1e.png)
     The pretrained model airbert.bin is downloaded from the link: https://github.com/airbert-vln/model-zoos, and the version is as below:
![image](https://user-images.githubusercontent.com/17741857/138737654-7edbd737-79a4-4e5f-9d10-055bc2997608.png)
      However, the result is not ideal. Finally, I just got a modal with SR=17.44 and SPL=14.60 on the unseen validation split. Have I missed ignored some details?  Or is the airbert.bin just the specified model  (r2rM_bnbMS_2capt.pth1.4.bin) in the original bash file?

**Comment by cshizhe** [comment]
This is weird. Could you share your training logs?
BTW, I released the exact r2rM_bnbMS_2capt.pth1.4.bin [here](https://www.dropbox.com/sh/wxcid5xjf8dvkt2/AAC4wPZw3UmqG1HfqGqGRHmBa?dl=0).

**Comment by hekj** [comment]
Hi, Shizhe

Thanks for your reply.

The training log based on airbert.bin is as below.
[train_log.txt](https://github.com/airbert-vln/airbert-recurrentvln/files/7422491/train_log.txt)
The log shows that the model gets the best performance (SR=17.44 and SPL=14.60 ) at 162000 iterations.

In addition, I have been training the REVERIE model based on the r2rM_bnbMS_2capt.pth1.4.bin. The model has gotten a performance with SR=20.08 and SPL=15.95 even only 15000 iterations were trained. This should be normal.

By the way, what is the difference between the airbert.bin and r2rM_bnbMS_2capt.pth1.4.bin? They seem to differ greatly in the performance of the REVERIE task.

**Comment by cshizhe** [comment]
We are checking the logs of r2rM_bnbMS_2capt.  
One thing you could check is if there are parameters not correctly initialized from the pretrained model.

**Comment by guhur** [comment]
Hi @hekj,

Thanks for pointing out this issue. 

I have trained `airbert.bin` and `r2rM_bnbMS_2capt.pth1.4.bin`; unfortunately, I can't be 100% confident for discriminating both checkpoints. The experiments are 10 months old, and I don't have logs anymore for them.

From the name convention, it seems that `r2rM_bnbMS_2capt.pth` means that the ViLBERT model was fine-tuned on R2R with a masking loss (`r2rM_`), then fine-tuned on the BNB dataset with the masking and shuffling loss (`bnbMS`) with the captionless image insertion strategy (a least 2 captioned images (`2capt`) are present).

So, I would say that `airbert.bin` has the three path-instruction augmented strategies: instruction rephrasing, image merging and captionless image insertion, whereas `r2rM_bnbMS_2capt.pth` would have been trained only with the captionless image insertion.
